### ISSUE TITLE
Staging solution for `alchemiscale.org` release migration

### ISSUE THREAD

**Issue by hmacdope** [asapdiscovery/asapdiscovery#1229]
@dotsdl would it be possible to work together to develop a strategy by which we can stage a transition from one alchemiscale.org patch version to another?

Currently our workflow is  
* wait for new alchemiscale.org release, 
* things are broken
* do urgent patch
* things work again

^ this is obviously poor practice in one or more respects.

Any suggested solutions to improve this would be great.

**Comment by hmacdope** [comment]
CC @JenkeScheen

**Comment by dotsdl** [comment]
Yes! I think this may dovetail well with my thoughts on a possible future state for all `alchemiscale.org` instances: https://github.com/OpenFreeEnergy/alchemiscale.org-deployment/issues/2

If we end up switching to a Kubernetes-based deployment of `alchemiscale.org` instances, it may be more possible to keep `qa` and `dev` instances alive at all times at lower cost than we do today, since currently we have these instances on dedicated EC2 hosts. This would then allow us to write a set of smoke tests that can be run with a Github Action against e.g. `qa` periodically and at any time to check that we do not observe breakage with the `asap-alchemy` tooling and the state of what will likely make its way to `prod`.

Alternatively, we could pull in some of the fixtures used in `alchemiscale` itself for its own integration tests into the `asap-alchemy` test suite, but these aren't the easiest to work with and may overcomplicate the `asap-alchemy` test suite.
